src/template-generator.js

```diff
@@ -0,0 +1,17 @@
+module.exports = (payload) => {
+    return `<html>
+    <head>
+        <title>Budget</title>
+    </head>
+    <body>
+        <div>
+            <h1>${payload.config.customerName}</h1>
```

Comment:
Let's to continue to use the handlebars because the report template will can be customized by pure html in this way I cannot remove then from javascript file

Reply:
Done